_layouts/index.html

```diff
@@ -18,32 +18,52 @@ <h1 class="logo">Helm</h1>
       <div id="boat"></div>
     </section>
 
+    <section class="panel -blue" data-purpose="news-and-updates">
+      <div class="news-and-updates">
+        <div class="news-and-updates__item container group">
+          <h2>Helm comes in two great flavors!</h2>
+          <article class="console">
+            <p>
+              The teams at Deis and Google are working hard to bring you a new and improved Helm.
+              In the meanwhile, the Deis team is committed to maintaining the original tool as
+              <i>Helm Classic</i>-- new name, same great taste! The name change ensures the original
```

Comment:
For maximum æsthetic foo, I'd go with the HTML entity `&mdash;` instead of `--`. And surround the em dash with a single space on each side, e.g.,

`<i>Helm Classic</i> &mdash; new name`...